core/src/main/scala/filodb.core/memstore/TimeSeriesShard.scala

```diff
@@ -1443,7 +1443,8 @@ class TimeSeriesShard(val ref: DatasetRef,
         val firstPartId = if (matches.isEmpty) None else Some(matches(0))
         val _schema = firstPartId.map(schemaIDFromPartID)
         val it1 = InMemPartitionIterator2(matches)
-        val partIdsToPage = it1.filter(_.earliestTime > chunkMethod.startTime).map(_.partID)
+        val partIdsToPage = debox.Buffer.empty[Int]
```

Comment:
NOTE: when the number of partitions is large, this may end up making much larger allocations than today.

Reply:
ack.